src/script/components/loading-button.ts

```diff
@@ -43,18 +81,18 @@ export class LoadingButton extends LitElement implements AppButtonElement {
 
   render() {
     return html`
-      <app-button
-        class=${this.className}
-        part="underlying-button"
-        .appearance=${this.appearance}
-        .type=${this.type}
-        .colorMode=${this.colorMode}
-        ?disabled=${this.disabled || this.loading}
+      <div 

```

Comment:
Same here as my above comment

Reply:
I applied the same changes as above here

Reply:
Sometimes the loading button is wrapped in a another button and sometimes it is not so to be safe I added the button role so either way it is acknowledged as a button but does not throw the nested interactive flag